projects/hitl/software/hitl/ft4222_proxy.py

```diff
@@ -0,0 +1,199 @@
+"""
+Proxy to access FT4222 Hardware. In normal use, these functions shouldn't be
+accessed directly, but instead through the IOController
+"""
+import logging
+import os
+
+from .utils import map_to_human, map_to_machine
+
+try:
+    import ft4222
+except ModuleNotFoundError:
+    # We haven't setup the ft4222 library yet...
+    # See https://awenstrup.github.io/setup.html
+    ft4222 = None
+
+# CONSTANTS
+# ADC parameters
+ADC_CHANNEL_TO_ADD_BITS = {
+    0: 0b00000000,
+    1: 0b00001000,
+    2: 0b00000001,
+    3: 0b00001001,
+}
+ADC_PREFIX = 0b10110000
+ADC_RETURN_SIZE_BYTES = 3
+
+# DAC parameters
+DAC_CHANNEL_TO_ADD_BITS = {i: i for i in range(8)}
```

Comment:
Is this and line 37 necessary? It was in the original code, but is there any reason we use the pin number directly in lines 84, 115, and 190?